src/oatpp-swagger/oas3/Model.hpp

```diff
@@ -294,6 +294,11 @@ class Schema : public oatpp::DTO {
    * Description of the field.
    */
   DTO_FIELD(String, description);
+  
+  /**
+   * Default value for the field.
+   */
+  DTO_FIELD(String, defaultValue);
```

Comment:
> I don't know how to get the representation of a Void defaultValue...

I think we can just change the type to `oatpp::Any` and everything should work `DTO_FIELD(Any, defaultValue)`